common/test/acceptance/pages/studio/settings_certificates.py

```diff
@@ -509,7 +509,6 @@ def delete_signatory(self):
         """
         Delete the signatory
         """
-        # pylint: disable=pointless-statement
```

Comment:
we got rid of this? I actually like this, I've caught code that has been rendered useless by refactoring w/ this one